packages/orchestration/src/orchestration-types.ts

```diff
@@ -446,13 +445,52 @@ export interface TranslationConfigParams {
   targetLanguage: string;
 }
 
+/**
+ * Output parameters for translation output configuration.
+ */
+export interface TranslationOutputParameters {
+  /**
+   * Language of the text to be translated.
+   * @example sourceLanguage: "de-DE"
+   */
+  sourceLanguage?: string;
+  /**
+   * Language to which the text should be translated.
+   * @example targetLanguage: "en-US"
+   */
+  targetLanguage: string;
+}
+
+/**
+ * Parameters for translation configurations.
+ */
+export type TranslationConfigParams<T extends 'input' | 'output'> =
```

Comment:
[q] Why not create a type for `'input' | 'output'` instead of duplicating it everywhere?

Reply:
True, anyway we don't export it. Good enough for me.